ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -657,6 +661,12 @@ nnfw_invoke (const GstTensorFilterProperties * prop,
   if (nnfw_status != NNFW_STATUS_NO_ERROR)
     err = -EINVAL;
 
+#if (DBG)
+  gint64 stop_time = g_get_real_time ();
+  g_message ("Invoke() is finished: %" G_GINT64_FORMAT,
+      (stop_time - start_time));
+#endif
+
```

Comment:
You are meauring incorrect values.

C-API Single Invoke Latency for nnfw = TOTAL-LATENCY(ml_single_invoke) - LATENCY(nnfw_run)

If we do memory_set in invoke, we'd better measure the latency of two memory_set calls as well. (need to see if these calls affect latency; if so, we need to find a way to mitigate them)

Reply:
Updated to measure latency added by single API + latency by the tensor filter extension.